### ISSUE TITLE
[Bug] Places cards are of variable height

### ISSUE THREAD

**Issue by mondalsurojit** [Omkar-Sonawane-23/Tourism#326]
**Describe the bug**
In the place pages, the card of each place is of variable heights, and thus has inconsistency in UI, making it somewhat less attractive.

**To Reproduce**
1. Go to 'Places' page
3. Scroll to see the variable heights of card

**Expected behavior**
All the cards should have been of same heights

**Screenshots**
![Screenshot (286)](https://github.com/Omkar-Sonawane-23/Tourism/assets/114282267/a7832903-3897-428f-89c7-98a956b86e99)
![Screenshot (287)](https://github.com/Omkar-Sonawane-23/Tourism/assets/114282267/dee6ef63-70a3-4eeb-a78f-44a0a5b0be38)

**Desktop (please complete the following information):**
 - OS: Windows 11 Home
 - Browser: Google Chrome
 - Device: Laptop

### Record

- [x] I agree to follow this project's Code of Conduct
- [x] I'm a GSSOC'24 contributor
- [X] I want to work on this issue

**Comment by sameer-soni** [comment]
@mondalsurojit assigned.

**Comment by sameer-soni** [comment]
@mondalsurojit are you working on this?

**Comment by mondalsurojit** [comment]
@sameer-soni Yes sir, I am working on it...Maybe I will do a pull request tomorrow, or at max day after tomorrow

**Comment by mondalsurojit** [comment]
@sameer-soni @Omkar-Sonawane-23 Sir please review my pr :)
